data/hai/hai missions.txt

```diff
@@ -1989,58 +2028,47 @@ mission "Strider 2 (Remnant)"
-			choice
-				`	"They're in need of your help."`
-				`	"They come seeking technology."`
-			`	A Remnant with a blood testing device has already approached the Hai. "We kindly request that you identify yourself," he chants. Yashili looks to you.`
-			choice
-				`	"They just want to test your blood to make sure you aren't an Alpha."`
-				`	"Don't worry, it doesn't hurt."`
-			`	The three Hai have their blood tested one by one. Instead of beeping and flashing yellow like when you took the blood test, the device instead flashes orange. But, when Yashili's blood is tested, the device makes a screeching noise instead of a beep and flashes red, prompting the Remnant to raise their weapons to Yashili. "Alpha!" one of them yells.`
-			choice
-				`	"No, she's not an Alpha! Test her again!"`
-					goto again
-				`	(Jump in front of Yashili.)`
-			`	You leap in front of Yashili and spread your arms as to be a bigger target. "Why do you protect this Alpha, outsider?" one of the Remnant asks.`
-			`	"She's not an Alpha, she's a Hai! Your devices must just not recognize alien blood properly. Test her again."`
-			
-			label again
-			`	The Remnant hesitate for a moment, but agree to drop their weapons and test Yashili again with a different device. This time the light flashes orange, just like for the other two Hai. "We will need to add these blood samples to our database," a Remnant says. "It would be unfortunate in the future if we harm an innocent being for fear of them being an Alpha."`
-			`	The delegation looks shaken after that confrontation, but Yashili has kept her composure and begins speaking. "We come representing the Hai people, who live far from here. Some of our brothers and sisters have turned violent toward us, and have recently created a new ship that may cause us trouble. We have created a new ship of our own in retaliation, but we are still in need of technology capable of repairing the hull of a ship while in flight so that we may have an advantage. Captain <last> has told us that you obtain such technology, with ship hulls that repair themselves."`
+			`At the designated co-ordinates there is a small encampment set up, looking vaguely reminiscent of desert traders from old movies. You note there are even a few camels tied near a water trough. When you touch down next to it, the delegation steps out with a confused look. "Captain, I thought you were taking us to where these people live?" Before you can answer, some Remnant approach. "Welcome to our encampment, representatives." Chant the Remnant. "Captain <first> brought us your missive, and we asked them to bring you to meet us here. Would you care to come inside?" They gesture towards the largest of the tents. While obviously surprised, Ambassador Yashili acquiesces and the Hai step inside. Three of the Remnant accompany them, and wave for you to follow. The remaining Remnant continue patrolling around the camp.`
```

Comment:
```suggestion
			`At the designated co-ordinates, there is a small encampment set up that looks vaguely reminiscent of desert traders from old movies. You note there are even a few camels tied near a water trough. When you touch down next to it, the delegation steps out with a confused look. "Captain, I thought you were taking us to where these people live. Is this the right place?" Before you can answer, some Remnant approach. "Welcome to our encampment, representatives," chant the Remnant. "Captain <first> brought us your missive, and we asked them to bring you to meet us here. Would you care to come inside?" They gesture toward the largest of the tents. While obviously surprised, Ambassador Yashili acquiesces, and the Hai step inside. Three of the Remnant accompany them and gesture for you to follow. The remaining Remnant continue patrolling around the camp.`
```
One sentence isn't a question, so I added a question at the end.
Some general grammar fixes.